.gitignore

```diff
@@ -41,7 +41,7 @@ DerivedData/
 !default.mode2v3
 *.perspectivev3
 !default.perspectivev3
-xcuserdata/
+# xcuserdata/
```

Comment:
What's the need to stop ignoring `xcuserdata`?
As far as I understand the only thing that is required to fix Carthage is the addition of SCNRecorder.xcscheme.

Reply:
Sorry, my mistake, remove unnecessary files.